.gitignore

```diff
@@ -12,3 +12,4 @@ ivy_lib/*
 .project
 .classpath
 .vs/*
+server/server.properties
```

Comment:
I don't see that server/server.properties.template file we talked about. Can you add that to the PR?